content/en/blog/_posts/2021-08-11-support-for-HostProcess-Containers/index.md.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: blog
+title: 'Windows HostProcess Containers'
+date: 2021-08-16
+slug: Windows HostProcess Containers
+---
+
+**Authors:** Brandon Smith
+
+{{<feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+HostProcess containers aim to extend the Windows container model to enable a wider 
+range of Kubernetes cluster management scenarios. HostProcess containers run 
+directly on the host and maintain behavior and access similar to that of a regular 
+process. With HostProcess containers, users can package and distribute management 
+operations and functionalities that require host access while retaining versioning 
+and deployment methods provided by containers. This allows Windows containers to 
+be used for a variety of device plugin, storage, and networking management scenarios 
+in Kubernetes. With this comes the enablement of host network mode - allowing 
```

Comment:
```suggestion
in Kubernetes. With this comes the enablement of host network mode--allowing 
```

Reply:
The sentence above should not use a hyphen, so I changed it to a dash. A comma might actually be better. Perhaps someone else has an opinion.